tests/math_translator_spec.js

```diff
@@ -52,8 +77,8 @@ describe('MathTranslator (translateMath)', function() {
     });
 
     it('should translate decimal point to decimal comma', function() {
-        const englishStr = '1000.000 + 9.4 + 45.0';
-        const translatedStr = '1000{,}000 + 9{,}4 + 45{,}0';
+        const englishStr = '900.4 + 45.0';
+        const translatedStr = '900{,}4 + 45{,}0';
```

Comment:
This should only test decimal comma, not thousand separator. Otherwise this test would fail for `ru`.